pkg/storaged/jobs-panel.jsx

```diff
@@ -39,50 +39,50 @@ const _ = cockpit.gettext;
  */
 
 const descriptions = {
-    'ata-smart-selftest':          _("SMART self-test of $target"),
-    'drive-eject':                 _("Ejecting $target"),
```

Comment:
These tabular views were actually kinda nice, but I suppose we'll have to bow to the machine linter bots.. :robot: